operators/Makefile

```diff
@@ -17,7 +17,7 @@ GKE_CLUSTER_VERSION ?= 1.12
 
 REPOSITORY 	?= eck
 NAME       	?= eck-operator
-VERSION    	?= 0.9.0-SNAPSHOT
+VERSION    	?= $(shell cat VERSION)-SNAPSHOT
```

Comment:
Why not also move `-SNAPSHOT` in the `VERSION` file?

Reply:
Fixed